packages/rs-drive-abci/tests/strategy_tests/strategy.rs

```diff
@@ -1104,17 +1106,37 @@ impl NetworkStrategy {
     fn create_identities_state_transitions(
         count: u16,
         key_count: KeyID,
+        extra_keys: &KeyMaps,
         signer: &mut SimpleSigner,
         rng: &mut StdRng,
         platform_version: &PlatformVersion,
     ) -> Vec<(Identity, StateTransition)> {
-        let (identities, keys) = Identity::random_identities_with_private_keys_with_rng::<Vec<_>>(
-            count,
-            key_count,
-            rng,
-            platform_version,
-        )
+        let (mut identities, mut keys) = Identity::random_identities_with_private_keys_with_rng::<
+            Vec<_>,
+        >(count, key_count, rng, platform_version)
         .expect("expected to create identities");
+
+        for identity in identities.iter_mut() {
```

Comment:
That's huge

Reply:
hehe I go where no programmer has gone before